### PR TITLE
Add message user button

### PR DESCRIPTION
**Pull request by luacmartins**
### Details
Adds a message user button when seeing a user's profile.

### Fixed Issues
$ https://github.com/Expensify/App/issues/6661

### Tests
1. Create a group chat.
2. Click the chat header and select one of the users.
3. Verify that there is a `Message <user>` button and clicking it navigates you to the chat with that user.

### QA Steps
Steps above.

### Tested On

- [X] Web
- [X] Mobile Web
- [x] Desktop
- [X] iOS
- [x] Android

#### Web

Screen Shot 2022-01-11 at 4 49 26 PM

#### Mobile Web

mweb

#### Desktop

desktop

#### iOS

ios

#### Android
android

### CODE REVIEW

**Comment by luacmartins** [comment]
@Expensify/design can I get your eyes on this one? Perhaps the spacing between `Local time` and `Message <user>` is too big?

**Comment by shawnborton** [comment]
I think it looks good personally, nice job!

**Comment by nickmurray47** [comment]
taking a look right now!

**Review by nickmurray47 — APPROVED** [review]
LGTM

**Comment by Beamanator** [comment]
possible regression? https://expensify.slack.com/archives/C01GTK53T8Q/p1642146678458100

File: src/pages/DetailsPage.js [hunk 1]
```diff
@@ -149,6 +152,13 @@ const DetailsPage = (props) => {
                                 </View>
                             ) : null}
                         </View>
+                        <MenuItem
+                            title={`${props.translate('common.message')}${details.displayName}`}
+                            icon={Expensicons.ChatBubble}
+                            onPress={() => Report.fetchOrCreateChatReport([props.session.email, details.login])}
+                            wrapperStyle={styles.breakAll}
+                            shouldShowRightIcon
+                        />
```
Comment (s77rt) [inline src/pages/DetailsPage.js:161]: This PR caused a regression https://github.com/Expensify/App/issues/18859. We should show the message button only if you are viewing another profile that is not yours. This was partially fixed by https://github.com/Expensify/App/pull/7228 but it did only check against the current session email and didn't cover the secondary logins.

**Pull request merged**